tpm/tpm.go

```diff
@@ -713,6 +851,103 @@ func OwnerReadSRK(rw io.ReadWriter, ownerAuth digest) ([]byte, error) {
 	return tpmutil.Pack(pk)
 }
 
+// ReadEKCert reads the EKCert from the NVRAM
+func ReadEKCert(rw io.ReadWriter, ownAuth digest) ([]byte, error) {
+	var (
```

Comment:
nit: remove this var block. you don't need it and can just use `:=` to allocate new variables.